src/wallet/init.cpp

```diff
@@ -55,6 +55,7 @@ void WalletInit::AddWalletOptions() const
 {
     gArgs.AddArg("-addresstype", strprintf("What type of addresses to use (\"legacy\", \"p2sh-segwit\", or \"bech32\", default: \"%s\")", FormatOutputType(DEFAULT_ADDRESS_TYPE)), false, OptionsCategory::WALLET);
     gArgs.AddArg("-avoidpartialspends", strprintf(_("Group outputs by address, selecting all or none, instead of selecting on a per-output basis. Privacy is improved as an address is only used once (unless someone sends to it after spending from it), but may result in slightly higher fees as suboptimal coin selection may result due to the added limitation (default: %u)"), DEFAULT_AVOIDPARTIALSPENDS), false, OptionsCategory::WALLET);
+    gArgs.AddArg("-avoidreuse", "Enable the 'avoid reuse' flag for new wallets created. When enabled, wallets will mark addresses which have been used to fund transactions in the past, and avoid reusing these in future funding, except when explicitly requested " + strprintf(_("(default: %u)"), DEFAULT_AVOIDREUSE), false, OptionsCategory::WALLET);
```

Comment:
My personal preference would be to remove this command line argument, since it's possible to create a wallet with this flag using the `createwallet` RPC, or update an existing wallet's flag with `setwalletflag`. I think it's better to remove it from here because:
- it's better to have one way to do things
- in general, I think we should be aiming to reduce the number of command line arguments, especially in the wallet (see https://github.com/bitcoin/bitcoin/issues/13044)

Reply:
Makes sense. And resolves one TODO (the startup option is ignored for "fresh" nodes in the test framework).